### PR TITLE
Daily buttons

### PR DESCRIPTION
**Pull request by Sybrand**
- Changed "daily" button to 5 different buttons for each day of prep.
- Clicking on button shows prep details for that day.
- Updated tests to contain date field.
- Added config for integrated debugging with chrome.
- Changed bug in contact mailto url
- Added check undefined/nan check on props.meanPrepLevel (resolves child is undefined error/warning)

# Test Links:
[Percentile Calculator](https://wps-pr-1653.apps.silver.devops.gov.bc.ca/)
[MoreCast](https://wps-pr-1653.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-1653.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-1653.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-1653.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=1108&f=c5&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN#state=2ec784ca-c46a-49d0-b2b3-1cf32a9015a2&session_state=7d9447c8-db66-4661-b4cb-03d2ac0d1d8f&code=32292df4-2bdf-4f90-a4a8-c8dbcda682a9.7d9447c8-db66-4661-b4cb-03d2ac0d1d8f.2b63f390-f3dc-43ae-89f2-016453863476)
[Fire Behaviour Advisory](https://wps-pr-1653.apps.silver.devops.gov.bc.ca/fire-behaviour-advisory)
[HFI Calculator](https://wps-pr-1653.apps.silver.devops.gov.bc.ca/hfi-calculator)

### CODE REVIEW

File: web/src/features/hfiCalculator/components/DailyViewTable.tsx [hunk 1]
```diff
@@ -326,9 +338,14 @@ export const DailyViewTable = (props: Props): JSX.Element => {
                         .sort((a, b) => (a[1].code < b[1].code ? -1 : 1))
                         .map(([stationCode, station]) => {
                           const daily = getDailiesByStationCode(
-                            1,
+                            numPrepDays,
                             props.dailies,
                             station.code
+                          ).filter(
```
Comment (Sybrand) [inline web/src/features/hfiCalculator/components/DailyViewTable.tsx:344]: I know there's some duplication here - but it's all dead code that the Conor PR should replace.

**Review by conbrad — APPROVED** [review]

File: .vscode/launch.json [hunk 2]
```diff
@@ -55,5 +55,12 @@
             "program": "openshift/s3-backup/docker/prune_test.py",
             "console": "integratedTerminal"
         },
+        {
+            "name": "Chrome",
+            "type": "pwa-chrome",
+            "request": "launch",
+            "url": "http://localhost:3000",
+            "webRoot": "${workspaceFolder}/web"
+        }
```
Comment (andrea-williams) [inline .vscode/launch.json:64]: what's the purpose of this addition?

**Review by andrea-williams — CHANGES_REQUESTED** [review]
Found a bug:

1. Wait until the page loads with data
2. Select one of the daily tabs
3. Change the date to any date and hit OK
4. When page eventually reloads, only the first day of prep period has data in the weekly table. Whomp-whomp.

File: .vscode/launch.json (hunk 2 above)
Comment (Sybrand) [reply .vscode/launch.json:64]: Integrated debugging with vscode ; I'll demo it in dev chat.

**Review by andrea-williams — APPROVED** [review]
Confirmed that the bug has been exterminated.

**Pull request merged**